pkg/types/types.go

```diff
@@ -83,6 +83,7 @@ type VulnerabilityDetail struct {
 	Description      string     `json:",omitempty"`
 	PublishedDate    *time.Time `json:",omitempty"` // Take from NVD or GHSA
 	LastModifiedDate *time.Time `json:",omitempty"` // Take from NVD or GHSA
+	Status           string     `json:",omitempty"` // Rejected or not
```

Comment:
We don't keep `Status` into `trivy-db` (IIUC this is correct behavior).

But It happens because `Normalize` function doesn't add `Status`:
https://github.com/aquasecurity/trivy-db/blob/fc85ecb02597bc72bb6c5ce28b6b53dcfa764450/pkg/vulnsrc/vulnerability/vulnerability.go#L44-L53

But i think we need to be clear about this. I see 2 ways:
1. Use ``json:"-"`` tag.
2. Add info in comment.

Reply:
Ah, yes, good point. We should not store that information in the database.